qemu/tests/cfg/cpu_add.cfg

```diff
@@ -115,10 +115,8 @@
             offline = 1
             vcpu_num_rechek = no
             ntp_server = "clock.redhat.com"
-            pre_command = "ntpdate ${ntp_server}"
-            ntp_sync_cmd = "ntpdate ${ntp_server}"
-            ntp_query_cmd = "ntpdate -q ${ntp_server}"
-            ntp_service_stop_cmd = "service chronyd stop || service ntpdate stop"
+            ntp_sync_cmd = "systemctl stop chronyd && chronyd -q 'server clock.redhat.com iburst'"
```

Comment:
We might need specify `service` command instead of `systemctl` for `RHEL.6` and `Host_RHEL.m6`.